portals/knowledge-management/src/Pages/Home.vue

```diff
@@ -68,6 +70,8 @@
     const loadingResults = ref(false);
     const searchResults = ref(null);
     const selectedTags = ref([]);
+    const route = useRoute();
+    const globalSearchInput = ref(null);
```

Comment:
I am really confused as to how this is working lol. But this isn't the correct type of `ref` I was talking about for this.

We should change to make sure we are using the right thing: https://vuejs.org/guide/essentials/template-refs.html

Reply:
Ah I see, this is a different type of way to do this. So many ways to do the same thing in Vue lol. All good